internal/setup/keyring.go

```
@@ -0,0 +1,331 @@
+)
+
+const (
+	keyringKeyFile     = "keyring.key"
+	credentialsEnvFile = "credentials.env"
+	passwordBytes      = 32 // 32 bytes = 64 hex chars
+
+	envKeyringPassword = "WK_KEYRING_PASSWORD" //nolint:gosec // env var name, not a credential
+	envKeyringBackend  = "WK_KEYRING_BACKEND"  //nolint:gosec // env var name, not a credential
+	envAccount         = "WK_ACCOUNT"
+
+	sourceLine = "source" // prefix used in shell profiles
```

golangci-lint (2.5.0)

[error] const sourceLine is unused

(unused)